lib/slog/slog.go

```diff
@@ -0,0 +1,125 @@
+// Package slog is a logger interface offering a uniformly unpleasant
+// and wearying experience for application developers, users and operators.
+//
+// TODO replace this entirely with something else. Maybe zerolog?
```

Comment:
This is done now, but I'd much rather you imported `log` or some external package instead of rolling your own. It doesn't add much to the challenge.